### ISSUE TITLE
Database is created but migrations don't run.

### ISSUE THREAD

**Issue by ianpurton** [luckyframework/lucky_migrator#100]
I can rub `db.create` and `db.drop` with no problems. when I connect to psql I can see the database has been created.

However running the following migration with 'lucky db.migrate' produces no output from the command and no tables are created.

class CreateUsers::V20180329061306 < LuckyMigrator::Migration::V1
  def migrate
    create :users do
      add email : String, index: true
      add encrypted_password : String
    end
  end

  def rollback
    drop :users
  end
end

Are the connection parameters used for creating the database different than the migrations?

**Comment by paulcsmith** [comment]
@ianpurton They should be the same connection parameters. It's definitely odd that no output comes form `lucky db.migrate`. It should say something, even if it's "Did nothing". Can you confirm that there is no output at all from the command? Also what folder is your migration in? Would it be possible for you to post a link to the repo so I can check it out?

Thanks so much for reporting!

**Comment by ianpurton** [comment]
I ran this again on a fresh c9 install. I tried OSX and that works fine so this is on Ubuntu.

Another guy had a similar issue and has made a pull request. https://github.com/luckyframework/lucky_migrator/pull/95

I think if you pull that in I would have got an error message.

So basically I do a lucky init and I run bin/setup

    $ lucky gen.migration CreateUsers
    Created CreateUsers::V20180413074552 in ./db/migrations/20180413074552_create_users.cr

I add the fileds as above.

I add one migration the one form the guide and I run 

    $ lucky db.migrate

Which just generates a blank line.

I'll do some more investigation.

**Comment by ianpurton** [comment]
OK, I solved the issue for me. C9 generetes a postgres user automatically called ubuntu. So by default lucky uses this user for it's connection URL.

This users requires a password when opening the database so doesn't work out of the box with lucky.

The reason lucky was able to create the database is that lucky runs the createdb command as a process which by default has access.

To solve this I created a lucky user for postgres and set the password. I then updated the DATABASE_URL to reflect this.
